RTEMS CFE_FT_Global build failure

**Describe the bug**

RTEMS CI fails due to "multiple definitions" of `CFE_FT_Global` 

```
[ 96%] Linking C shared module cfe_testcase.obj
/root/rtems-5/bin/i386-rtems5-ld: CMakeFiles/cfe_testcase.dir/src/cfe_test.c.o:(.bss+0x0): multiple definition of `CFE_FT_Global'; CMakeFiles/cfe_testcase.dir/src/cfe_test_table.c.o:(.data+0x0): first defined here
/root/rtems-5/bin/i386-rtems5-ld: CMakeFiles/cfe_testcase.dir/src/tbl_content_access_test.c.o:(.bss+0x0): multiple definition of `CFE_FT_Global'; CMakeFiles/cfe_testcase.dir/src/cfe_test_table.c.o:(.data+0x0): first defined here
/root/rtems-5/bin/i386-rtems5-ld: CMakeFiles/cfe_testcase.dir/src/tbl_content_mang_test.c.o:(.bss+0x0): multiple definition of `CFE_FT_Global'; CMakeFiles/cfe_testcase.dir/src/cfe_test_table.c.o:(.data+0x0): first defined here
/root/rtems-5/bin/i386-rtems5-ld: CMakeFiles/cfe_testcase.dir/src/tbl_information_test.c.o:(.bss+0x0): multiple definition of `CFE_FT_Global'; CMakeFiles/cfe_testcase.dir/src/cfe_test_table.c.o:(.data+0x0): first defined here
/root/rtems-5/bin/i386-rtems5-ld: CMakeFiles/cfe_testcase.dir/src/tbl_registration_test.c.o:(.bss+0x0): multiple definition of `CFE_FT_Global'; CMakeFiles/cfe_testcase.dir/src/cfe_test_table.c.o:(.data+0x0): first defined here
```

**To Reproduce**

Checkout cFS bundle at `871a2d7` then point cFE to 600fd7a320e0a11cd892bf311d490735ca132f5a

**Expected behavior**
Both Linux and RTEMS build succesfully

**Code snips**
Related to setting Table releated entries in global struct in `modules/cfe_testcase/src/cfe_test_table.c`

```c
/* Constant Table information used by all table tests */
CFE_FT_Global_t CFE_FT_Global = {
    .TblName = "TestTable", .RegisteredTblName = "CFE_TEST_APP.TestTable", .TblFilename = "test_tbl.tbl"};
```

**System observed on:**
CI build of RTEMS 4.11 and 5.0

**Additional context**
Found during IC:2021-08-10 process. 

See https://github.com/nasa/cfe/issues/1772 and https://github.com/nasa/cFS/pull/333

--- a/modules/cfe_testcase/CMakeLists.txt
+++ b/modules/cfe_testcase/CMakeLists.txt
@@ -2,9 +2,9 @@ include_directories(inc)
 
 # Filenames based on doxygen groups.
 # Create the app module
-add_cfe_app(cfe_testcase
-    src/cfe_test_table.c
+add_cfe_app(cfe_testcase    
     src/cfe_test.c
+    src/cfe_test_table.c
     src/es_application_control_test.c
     src/es_info_test.c
     src/es_task_test.c

--- a/modules/cfe_testcase/src/cfe_test.c
+++ b/modules/cfe_testcase/src/cfe_test.c
@@ -41,6 +41,11 @@ CFE_FT_Global_t CFE_FT_Global;
  */
 void CFE_TestMain(void)
 {
+    /* Constant Table information used by all table tests */
+    CFE_FT_Global.TblName           = "TestTable";
+    CFE_FT_Global.RegisteredTblName = "CFE_TEST_APP.TestTable";
+    CFE_FT_Global.TblFilename       = "test_tbl.tbl";
+
     /*
      * Register this test app with CFE assert
      *

--- a/modules/cfe_testcase/src/cfe_test_table.c
+++ b/modules/cfe_testcase/src/cfe_test_table.c
@@ -33,13 +33,10 @@
 #include "cfe_test.h"
 #include "cfe_test_table.h"
 
-/* Constant Table information used by all table tests */
-CFE_FT_Global_t CFE_FT_Global = {
-    .TblName = "TestTable", .RegisteredTblName = "CFE_TEST_APP.TestTable", .TblFilename = "test_tbl.tbl"};
-
 /* Setup function to register a table */
 void RegisterTestTable(void)
 {
+
     UtAssert_INT32_EQ(CFE_TBL_Register(&CFE_FT_Global.TblHandle, CFE_FT_Global.TblName, sizeof(TBL_TEST_Table_t),
                                        CFE_TBL_OPT_DEFAULT, NULL),
                       CFE_SUCCESS);

--- a/modules/cfe_testcase/src/cfe_test_table.h
+++ b/modules/cfe_testcase/src/cfe_test_table.h
@@ -39,8 +39,6 @@
  */
 #include "cfe_test.h"
 
-CFE_FT_Global_t CFE_FT_Global;
-
 void RegisterTestTable(void);
 void UnregisterTestTable(void);
 

